src/main/java/jacchm/footballapp/exception/ExternalFootballApiConnectionException.java

```diff
@@ -0,0 +1,12 @@
+package jacchm.footballapp.exception;
+
+public class ExternalFootballApiConnectionException extends RuntimeException {
+
+    private final String errCode;
+
+    public ExternalFootballApiConnectionException(String message, String errCode) {

```

Comment:
Myślę, żeby nie dorabiać sobie roboty, szczególnie, że używasz podejścia z errCode, zamiast dwóch własnych takich samych customowych wyjątków lepiej zrobić jeden - np. FootballAppException, a problem czy to był błąd parsowania czy błąd Parsowania zrobić przez errCode np. ERR0001 to problem z parsowaniem a ERR0002 to problem z zewnętrznym API. Dodatkowo errCode koniecznie jako Enum a nie zwykły String, bo to jakby są wyjątki których się spodziewasz i którym nadajesz kody, więc ich zawsze będzie skończona ilość.